MetricsUploader/TestClients/SendEventClient.cpp

```diff
@@ -0,0 +1,17 @@
+// Copyright (c) 2020 The Orbit Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#include <cstdint>
+
+#include "MetricsUploader/Result.h"
+
+#define DllExport __declspec(dllexport)
```

Comment:
Currently there is no reason for having this macro. It's usually there because you surround it by `#ifdef _WIN32` and `#endif`.

So I'd either add the `ifdef` or remove it.

Reply:
Removed `#define ` here